test/config_test/config_test.cc

```diff
@@ -75,7 +77,9 @@ class ConfigTest {
 
 uint32_t run(const std::string& directory) {
   uint32_t num_tested = 0;
-  for (const std::string& filename : TestUtility::listFiles(directory, true)) {
+  // Change working directory, otherwise we won't be able to read files using relative paths.
+  RELEASE_ASSERT(::chdir(directory.c_str()) == 0);
```

Comment:
Per above we need this to still work with cert files that can't be read. Sorry. :(